### ISSUE TITLE
Support for ARM64 Architecture for cal.com Docker Image on Docker Hub

### ISSUE THREAD

**Issue by hostsistemas** [calcom/cal.com#15315]
Currently, the Docker Hub repository for cal.com only supports the AMD64 architecture. As ARM64 architecture is becoming increasingly popular, especially with the rise of ARM-based servers and devices such as the Apple M1, it would be beneficial to extend support to include ARM64.
 
Please provide support for the ARM64 architecture for the cal.com Docker image available on Docker Hub. This will allow users with ARM-based systems to run the Docker image natively, improving performance and compatibility.

**Comment by dandry** [comment]
+1 for this.
Would be great to see arm64 compatibility. 
Thanks.

**Comment by fabio0296** [comment]
+1 for this as well.
But shouldn't this issue be created on https://github.com/calcom/docker ? I think i'd get vissibilty there.

**Comment by shsunmoonlee** [comment]
I'm facing the issue and addressed this on docker repo. https://github.com/calcom/docker/issues/358#issuecomment-2544588212

**Comment by jeveloper** [comment]
Id like to participate in this and help bring it to market.

**Comment by neoighodaro** [comment]
Same issue here as well

**Comment by jeveloper** [comment]
I wouldn't mind connecting with any contributor to get a sense of "ideal outcome". I want to ask a few questions but keep it detailed and concise. I don't like to assume things.

**Comment by marcosmartinezfco** [comment]
Yep same issue here trying to deploy to my k8s cluster using spot arm instances

**Comment by codewhizz** [comment]
+1 for this.

**Comment by NiceATC** [comment]
+1

**Comment by btwitsPratyush** [comment]
Hi! 👋  
I’d love to work on this issue. Let me know if it's available and if there are any guidelines I should follow. Thanks!

**Comment by jeveloper** [comment]
> Hi! 👋 I’d love to work on this issue. Let me know if it's available and if there are any guidelines I should follow. Thanks!

Hey, question for you. I plan to spend a few week of my free and commercial time, what would be a reasonable reward you'd be willing to pay for it? 

Given I still contribute to the open sourced tree here.

fair?

**Comment by tigerblue77** [comment]
Seems fixed https://hub.docker.com/r/calcom/cal.com/tags

**Comment by jeveloper** [comment]
@tigerblue77  there is a docker imagine now , you're right. Perhaps can be optimized more.

**Comment by tigerblue77** [comment]
Optimized by keeping amd64 as the "latest" image and not making the arm image the default ? Yes. https://github.com/calcom/cal.com/issues/25182
Sorry, nothing against you, just this application is a big mess, almost 1K issues, I wonder WTF are they doing. And v5.9.7 not working.... I'm tired of it. Really.

**Comment by PeerRich** [comment]
> just this application is a big mess, almost 1K issues, I wonder WTF are they doing. And v5.9.7 not working.... I'm tired of it. Really.

we have 35 team members on staff wroking day and night through the code base thats being used by millions of people

at any given moment we have 370 open PRs that we work through. more analytics here: cal.com/open

shame is easy, provide fixes. this is open source baby

**Comment by PeerRich** [comment]
seems fixed btw. please reopen if still broken

**Comment by tigerblue77** [comment]
> we have 35 team members on staff wroking day and night through the code base thats being used by millions of people
> at any given moment we have 370 open PRs that we work through. more analytics here: cal.com/open

Well, you're basically just saying you're understaffed, and that's okay as long as you're doing your best to fix that. And I'm concerned for clients & users, not blaming for free.

> shame is easy

Sure

> provide fixes

I help where I can on issues, don't count on me to propose code for a commercial application/not fully open source

> this is open source baby

Wanted to apologize for my previous exhausted message but, seeing this disrespect, I close the discussion as is.

**Comment by jeveloper** [comment]
Congrats to team releasing 6.x version , holy moly so many things added and improved, but also thats a lot. Hope the team took a long enough break.

and thanks for folks helping on the OSS side.
